src/main/java/echoserver/EchoClient.java

```diff
@@ -0,0 +1,45 @@
+package echoserver;
+import java.io.*;
+import java.net.*;
+
+public class EchoClient {
+    private static int port = 8080;
+    private static String host;
+    private static Socket clientSocket;
+
+    public static void main(String[] args) {
```

Comment:
Does the EchoClient need its own run task/cmd? Or are we only ever interacting with it through the EchoServer run task?